docs/pages/guides/pages/signin.mdx

```diff
@@ -192,6 +192,62 @@ export default async function SignInPage() {
 ```
 
 </Code.Next>
+<Code.NextClient>
+
+```tsx filename="app/signin/page.tsx"
+"use client"
+import { AuthError } from "next-auth"
+import { signIn } from "next-auth/react"
+import { providerMap } from "@/auth"
```

Comment:
this is imported from a file with potentially server-side code, so most likely will break. <picture data-single-emoji=":sad-cowboy:" title=":sad-cowboy:"><img class="emoji" width="20" height="auto" src="https://emoji.slack-edge.com/T0CAQ00TU/sad-cowboy/b8413a8862df8f19.png" alt=":sad-cowboy:" align="absmiddle"></picture> I think to support this case, we would need to recommend a config split, which I don't think is worth it just to support client-side login pages, which don't really have much of a benefit anyway.

Reply:
That makes sense, it's similar to what was discussed in issue #11803. I might join the Discord and look into making some contributions as well.